website/src/client/components/DevicePreview/WebFrame.tsx

```diff
@@ -41,3 +41,43 @@ const styles = StyleSheet.create({
     backgroundColor: c('content', 'light'),
   },
 });
+
+/**
+ * List of all iFrame permission directives.
+ * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy#directives
+ */
+const iframePermissions = [
+  'accelerometer',
+  'ambient-light-sensor',
+  'autoplay',
+  'battery',
+  'camera',
+  // 'display-capture',
+  // 'document-domain',
+  // 'encrypted-media',
+  // 'execution-while-not-rendered',
+  // 'execution-while-out-of-viewport',
```

Comment:
Can we remove the disabled rules instead? There is a link in comment already, which leads to the full list of possible directive.